spec/requests/users_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+describe UsersController, type: :request do
+  before do
+    @user = User.create(name: 'Sam', photo: 'photo.url', bio: 'Excellent', posts_counter: 0)
+  end
+
+  describe 'GET #index' do
+    it 'responds with success' do
+      get '/users'
+      expect(response).to have_http_status(:success)
+    end
+
+    it 'renders the index template' do
+      get '/users'
+      expect(response).to render_template('users/index')
+      expect(response).to render_template(layout: 'layouts/application')
+    end
+
+    it 'assigns @users with all users' do
+      get '/users'
+      expect(assigns(:users)).to eq(User.all)
+    end
+  end
```

Comment:
- [ ] According to the requirements:

![image](https://github.com/uno36/blog-app-rails/assets/77069011/d32d1289-f3d2-461a-bc1c-502a2ff019e7)

Please include a test for the placeholder text `response.body` 🤓 🔧